internal/repo/client/metaclient.go

```diff
@@ -101,6 +103,6 @@ func (d metaclient) GetReposForPackage(name string) ([]v1alpha1.PackageRepositor
 				}
 			}
 		}
-		return result, compositeErr
+		return result, repoerror.Partial(compositeErr)
```

Comment:
I think we could go one step further and only return a partial error if at least one repo succeeded (and at least one failed).